docs/configuration.rst

```diff
@@ -281,20 +95,20 @@ Removes a given reference set from a given data repository.
 
 .. code-block:: bash
 
-    $ ga4gh_repo remove-referenceset path/to/datarepo aReferenceSet
+    $ ga4gh_repo remove-referenceset path/to/repo.db aReferenceSet
 
 ++++++++++++++++
-add-ontologymap
+add-ontology
```

Comment:
This has gone back and forth once or twice, just making sure we land where want to be "map" or not "map".